tests/Rules/UselessConstructs/NoUselessValueFunctionCallsRuleTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rules\UselessConstructs;
+
+use Tests\RulesTest;
+
+class NoUselessValueFunctionCallsRuleTest extends RulesTest
```

Comment:
Please extend `RuleTestCase` here instead of `RulesTest` And change the rest of the class according to it.